src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
```

Comment:
Please consider using inheritance to abstract out the Todo, Deadline and Event as they share different key properties, it will be much easier to code out later levels when you have to use LocalDate!

Reply:
okay, thanks for the tip! I've since incorporated inheritance into my code